bloom-housing/bloom
# feat: programs on listings

This PR addresses #4399 


## Description
Adds programs to the listing detail page

## How Can This Be Tested/Reviewed?
by default the seeded data has programs set to show on the listing detail page of the public site 

head to the public site select a listing with programs on it and you should get the new program section

make sure translations work for each of the languages

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
